### ISSUE TITLE
Disambiguate with relative paths

### ISSUE THREAD

**Issue by lovette** [midnightprioriem/mkdocs-autolinks-plugin#39]
I have a structure similar to this (but much larger with dozens of duplicate file names.)

	- runtime/
	  - index.md
	  - foo.md
	- arch/
	  - index.md
	  - foo.md

I was hoping each `index.md` could cross link to `foo.md` with relative paths like `./foo.md` or `runtime/foo.md` and `arch/foo.md` but it appears that doesn't work. I still have to use `../runtime/foo.md`, etc. 

Would be a nice feature!

**Comment by stephffuller** [comment]
Yes, very much agreed! Using the index.md filename is mandated for every folder in mkdocs if you want the folder name to link to a page rather than just be a dead heading. After adding autolinks plugin, I'm getting dozens of errors about duplicate index.md files. But I cannot rename them.
